### PR TITLE
Update Sass documentation

### PR DESCRIPTION
**Pull request by andysellick**
- following change to sass imports for individual components

## What / why
We've got a new way of importing sass for components. This change updates the documentation to reflect that.

## Visual Changes
None.

### CODE REVIEW

File: docs/install-and-use.md [hunk 1]
```diff
@@ -53,6 +53,24 @@ To make use of the components, you'll have to include the JS and CSS in your app
 
 ### CSS
 
+The Sass for any components in use in an application should be imported. In order for this to work you must first import some generic Sass files that include `govuk-frontend` and some gem specific mixins and variables.
+
+Note that you will need to keep this up to date if you add or remove components from your application.
+
+For example:
+
+```
```
Comment (injms) [inline docs/install-and-use.md]:
```suggestion
```scss
```

Comment (andysellick) [reply docs/install-and-use.md]: Thanks! Done. Also you win the prize for 'fastest unsolicited review ever' 😁

File: docs/install-and-use.md [hunk 2]
```diff
@@ -53,15 +53,33 @@ To make use of the components, you'll have to include the JS and CSS in your app
 
 ### CSS
 
-```css
-# application.scss
+The Sass for any components in use in an application should be imported. In order for this to work you must first import some generic Sass files that include `govuk-frontend` and some gem specific mixins and variables.
+
+Note that you will need to keep this up to date if you add or remove components from your application.
```
Comment (injms) [inline docs/install-and-use.md]: Worth bolding?

```suggestion
**Note that you will need to keep this up to date if you add or remove components from your application.**
```

File: docs/install-and-use.md [hunk 3]
```diff
@@ -53,15 +53,33 @@ To make use of the components, you'll have to include the JS and CSS in your app
...
+
+```scss
+// supporting sass
+@import 'govuk_publishing_components/govuk_frontend_support';
+@import 'govuk_publishing_components/component_support';
+// component specific sass
+@import 'govuk_publishing_components/components/_back-link';
+```
+
+If you load the component guide in the application the suggested Sass for this application has been included for you. Click the 'Suggested Sass for this application' link, then copy and paste the output into your `application.scss` file. Remember to also copy the suggested print Sass into your application's `print.scss` file.
```
Comment (injms) [inline docs/install-and-use.md]: Repetition of 'application'

```suggestion
If you load the component guide in the application the suggested Sass for has been included for you. Click the 'Suggested Sass for this application' link, then copy and paste the output into your `application.scss` file. Remember to also copy the suggested print Sass into your application's `print.scss` file.
```

File: docs/install-and-use.md [hunk 4]
```diff
@@ -53,15 +53,33 @@ To make use of the components, you'll have to include the JS and CSS in your app
...
+// supporting sass
+@import 'govuk_publishing_components/govuk_frontend_support';
+@import 'govuk_publishing_components/component_support';
+// component specific sass
+@import 'govuk_publishing_components/components/_back-link';
+```
+
+If you load the component guide in the application the suggested Sass for it has been generated for you. Click the 'Suggested Sass for this application' link, then copy and paste the output into your `application.scss` file. Remember to also copy the suggested print Sass into your application's `print.scss` file.
+
+### CSS (deprecated, will be removed in a later version)
```
Comment (injms) [inline docs/install-and-use.md]: There are two similar headings' - could we make the difference between the two import methods clearer?

```suggestion
### Import all CSS

>> This is deprecated and will be removed in a later version
```

File: docs/install-and-use.md [hunk 5]
```diff
@@ -53,15 +53,33 @@ To make use of the components, you'll have to include the JS and CSS in your app
 
 ### CSS
```
Comment (injms) [inline docs/install-and-use.md]: Update to make the different methods clearer?

```suggestion
### Import component CSS
```

**Comment by kr8n3r** [comment]
@injms is on fire with suggestions, looks good to me

**Pull request merged**